microservice/questionService/index.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config()
+
+// Set up
+const express = require('express')
+const mongoose = require('mongoose')
+
+const app = express()
+
+app.use(express.json())
+
+const port = process.env.PORT || 3000
```

Comment:
Since multiple processes cannot use the same ports, maybe we could use port 3002 here?

3000: Front-end 
3001: User Service
3002: Questions Service etc

Reply:
Updated!